index.html

```diff
@@ -47,6 +44,15 @@ <h1>IHS Housing Stock Map</h1>
 
     </nav>
     <main role="main" class="container-fluid">
+        <div id="area-info" class="ps-2">
+            <p class="mb-0" id="placeholder">
+                <i class="fa fa-hand-pointer-o" aria-hidden="true"></i> Hover or select an area for details.
+            </p>
+            <div id="info-display" style="display: none;">
+                <div><span class="me-2">Name:</span><strong id="name"></strong></div>
+                <div><span class="me-2">Total Units:</span><strong id="units"></strong></div>
```

Comment:
lets have the hover show the colors and legend, along with percentages for each category. we can continue to show both sets of numbers in the click